### PR TITLE
Parse response score as Double instead of Float to resolve permanent loading

### LINKED ISSUE #22: Bug in ViewController.swift JSON communication
If you run the current code you have to change https://github.com/IBM/watson-waste-sorter/blob/b49b8e51b52a1ebfc993c48b60f247e880f54d53/mobile-app/WatsonWasteSorter/ViewController.swift#L56

from

 if let confs = JSON["confident score"] as? Float {

to

if let confs = JSON["confident score"] as? Double{

The reason of this bug is maybe a change on the server side, but this will fix it on the client side.

### PR DESCRIPTION
**Pull request by maggix**
Resolves #22

### CODE REVIEW

**Review by scottdangelo — APPROVED** [review]

File: server/run.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def sort():
                         "confident score": 0})
         parameters = json.dumps({'classifier_ids': [classifier_id]})
         url_result = visual_recognition.classify(images_file=images_file,
-                                                 parameters=parameters)
+                                                 parameters=parameters).get_result()
```
Comment (Tomcli) [inline server/run.py]: Thanks for pull request, can you move this line under 79 characters to follow the flake8 guideline? Thank you.

**Comment by Tomcli** [comment]
@maggix To follow the flake8 guideline, can you modify `server/run.py`'s line 63 to 64 to the following? Thank you.
```python
        url_result = visual_recognition.classify(
                        images_file=images_file,
                        parameters=parameters).get_result()
```

**Review by stevemar — APPROVED** [review]
@Tomcli give this one last review please

**Review by Tomcli — APPROVED** [review]
LGTM

**Pull request merged**